website_breadcrumb/views/templates.xml

```diff
@@ -6,6 +6,13 @@
 <data>
 
 <template id="breadcrumb" name="Breadcrumb snippet">
+    <t t-call="website_breadcrumb.breadcrumb-generator">
+        <t t-set='include_self' t-value='False'/>
+        <t t-set='revert_order' t-value='True'/>
+    </t>
+</template>
+
+<template id="breadcrumb-generator" name="Breadcrumb snippet generator">
```

Comment:
Hi @Yajo 
IMO the dependency order is correct like that too because `website_breadcrumb` provides the base and `website_snippet_breadcrumb` provides an added value. I suggest to overwrite the template into the `website_snippet_breadcrumb` in order to disable it by default. By this way we switch into manual mode and only care about the dynamic snippet.
This could be a quick solution what do you think?